packages/manager/apps/container/src/container/common/nav-reshuffle-switch-back/index.tsx

```diff
@@ -21,20 +19,17 @@ function NavReshuffleSwitchBack(): JSX.Element {
   const { updateBetaChoice, betaVersion, useBeta } = useContainer();
   const shell = useShell();
   const trackingPlugin = shell.getPlugin('tracking');
-  const isSmallDevice = useMediaQuery({
-    query: `(max-width: ${SMALL_DEVICE_MAX_SIZE})`,
-  });
   const [confirm, setConfirm] = useState<boolean>(false);
   const onboarding = useOnboarding();
 
-  if (!betaVersion || isSmallDevice) {
+  if (!betaVersion) {
     return <></>;
   }
 
   const toggleVersion = (value: 'classic' | 'beta') => {
     const versionName = value === 'beta' ? 'new' : 'old';
     trackingPlugin.trackClick({
-      name: `topnav::switch_version::go_to_${versionName}_version`,
+      name: `topnav::switch_version_V3::go_to_${versionName}_version`,
```

Comment:
Are we sure we keep `topnav::` here ?
_It seems to be removed elsewhere (like in packages/manager/apps/container/src/container/common/nav-reshuffle-switch-back/Modal.tsx)_

Reply:
Indeed, it is the Tracking team reco